packages/types-algosdk/types/index.d.ts

```diff
@@ -1,770 +0,0 @@
-/* eslint-disable  @typescript-eslint/naming-convention */
```

Comment:
Should we delete this immediately? If a user is using this package (say via linking), then code will break after release. Maybe we should add a depreciation notice first?

Reply:
we are in develop branch, so this is OK. we can add this in master before updating.